client/my-sites/plans/ecommerce-trial/ecommerce-trial-banner/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Card } from '@automattic/components';
+import { useLocale } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import {
+	getCurrentPlan,
+	getECommerceTrialDaysLeft,
+	getECommerceTrialExpiration,
+	isECommerceTrialExpired,
+} from 'calypso/state/sites/plans/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import DoughnutChart from '../../doughnut-chart';
+
+import './style.scss';
+
+const ECommerceTrialBanner = () => {
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) ) || -1;
+
+	const { currentPlan, eCommerceTrialDaysLeft, isTrialExpired, eCommerceTrialExpiration } =
+		useSelector( ( state ) => ( {
+			currentPlan: getCurrentPlan( state, selectedSiteId ),
+			isTrialExpired: isECommerceTrialExpired( state, selectedSiteId ),
+			eCommerceTrialDaysLeft: Math.round( getECommerceTrialDaysLeft( state, selectedSiteId ) || 0 ),
+			eCommerceTrialExpiration: getECommerceTrialExpiration( state, selectedSiteId ),
+		} ) );
+
+	const locale = useLocale();
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	const trialStart = moment( currentPlan?.subscribedDate );
+	const trialEnd = moment( currentPlan?.expiryDate );
```

Comment:
I think we should be using `moment.utc()` here, as these times are in UTC, not local time.